script.js

```diff
@@ -1,12 +1,207 @@
-//You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+//Variables
+// Div for showing episodes or movies
+const showEpisodseRow = document.querySelector(".row");
+// Search input
+const searchUser = document.querySelector("#searchUser");
+// Show the number of episodes based on search
+const searchNumResult = document.querySelector(".epi-num");
+// Select related to shows
+const selectShows = document.querySelector("#select-shows");
+// Select related to episodes
+const selectEpisodes = document.querySelector("#select-episodes");
+// get all shows
+const allShows = getAllShows();
+let allEpisodes;
+
+
+//EventListeners
+// display episodes on load
+document.addEventListener("DOMContentLoaded", setup);
+// Search specific Show based on select
+selectShows.addEventListener("change", selectShow);
+// Search specific episode based on select
+selectEpisodes.addEventListener("change", selectEpisode);
+// Search episodes
+searchUser.addEventListener("keyup", searchMovie);
+
+
+//Functions
+// Get all shows
+async function setup() {
+  // Make card for all shows and create options for select
+  makePageForShows(allShows);
 }
 
+// Make card for all shows
+function makePageForShows(allShows) {
+  searchUser.disabled = true;
+  allShows.sort((a, b) =>
+    a.name.toLowerCase().localeCompare(b.name.toLowerCase())
+  );
+  let output = "";
+  allShows.forEach((show) => {
+    output += `
+      <div class="col" style="height: 550px">
+        <div class="card h-100 px-0 mx-1">
+          <img src="${
+            show.image !== null ? show.image.medium : ""
+          }" style="height: 250px" class="card-img-top mx-0" alt="...">
+          <div class="card-body mb-4">
+          <h5 class="text-primary">${show.name}</h5>
+          <span class="badge text-bg-dark">Rated: ${show.rating.average}</span>
+          <span class="badge text-bg-dark">Genres: ${show.genres}</span>
+          <span class="badge text-bg-dark">Status: ${show.status}</span>
+          <span class="badge text-bg-dark">Runtime: ${show.runtime}</span>
+          <p>${
+            show.summary.split(" ").length >= 25
```

Comment:
as with episode summaries, I'd recommend splitting this out into its own logic for dealing with summaries, and reusing it between shows and episodes.